manager.py

```diff
@@ -48,4 +49,4 @@ def create_directory(path = "/"):
 
 
 if __name__ == '__main__':
-    app.run(host="0.0.0.0")
+    app.run(host="127.0.0.1", port=8080) # for local testing
```

Comment:
@maxtortime could you please at least leave port setting as default. I don't think it's critical to this PR.